### DISCUSSION TITLE
Real-Time Data Integration from Twenty via API

### DISCUSSION THREAD

**Discussion by kannan-immo** [twentyhq/twenty#7882] [Engineering vision]
Does Twenty have the capability to send data created or updated in the app to other applications through an API? We have a use case where we need to capture data entered in Twenty and use it in another app for further processing. Specifically, we’re looking for a solution that allows us to capture data from Twenty in real time.

**Comment by BOHEUS** [comment]
@kannan-immo you can do it using webhooks as they're triggered upon creating, updating or deleting object/record, one way to achieve this is using middleware like n8n or Windmill where workflow starts from Twenty webhook and sends data to your app
